app/models/person.rb

```diff
@@ -247,7 +224,7 @@ def update_remote_attributes(remote_attributes)
   private
 
   def update_nation_builder
-    relevant_fields = changed & FIELDS_ALSO_ON_NB
+    relevant_fields = changed & %w[email first_name last_name is_volunteer phone]
```

Comment:
Why not use `Integration::NationBuilder::PERMITTED_PERSON_PARAMS` or some other constant from NationBuilder class here.  

Also, I wonder if we could push the responsibility for this logic in a more appropriate place.  Maybe it should live in  `PersonWithRemoteFields`? Ideally, we could just call NbPersonPushJob or similar, and that job's preform would decide what to do with itself.